examples/vanilla-tweaks/docker-compose.yml

```diff
@@ -25,3 +25,19 @@ services:
       VERSION: ${MINECRAFT_VERSION:-LATEST}
       VANILLATWEAKS_SHARECODE: MGr52E
       REMOVE_OLD_VANILLATWEAKS: "TRUE"
+  vanillatweaks_file_datapacks_and_resourcepacks_and_craftingtweaks:
```

Comment:
I'm not sure if I had noticed before this example file was one big list of services 😀